tt_fabric/mesh_graph_descriptors/tg_mesh_graph_descriptor.yaml

```diff
@@ -0,0 +1,66 @@
+ChipSpec: {
+  arch: wormhole_b0,
+  ethernet_ports: {
+    N: 4,
+    E: 4,
+    S: 4,
+    W: 4,
+  }
+}
+
+
+Board: [
+  { name: Galaxy,
+    type: Mesh,
+    topology: [4, 8]},
+  { name: N150Gateway,
+    type: Mesh,
+    topology: [1, 1]}
+]
+
+Mesh: [
+{
+  id: 0,
+  board: N150Gateway,
+  topology: [1, 1],
+  host_mapping: [[aus-glx-11]]},
```

Comment:
Is this tg descriptor tied to aus-glx-11?
I suppose control plane is not currently using this field, so it does not matter...